tests/Application.cfc

```diff
@@ -70,7 +70,7 @@ component {
 			// Custom Module Entities
 			rootPath & "modules_app"
 		],
-		dialect               : "org.hibernate.dialect.MySQL5InnoDBDialect", // MySQL Dialect
+		dialect               : "org.hibernate.dialect.MySQL8Dialect", // MySQL Dialect
```

Comment:
This is problematic @michaelborn because this doesn't exist in previous hibernate versions, you can see that the tests failed on adobe engines.  So I am not thinking we can do this just yet.

Reply:
Interesting. Well, the MySQL 5 dialect won't work on Lucee with MySQL 8. We should be able to use the dialect shorthand - like `MySQL`. I will give that a try and see if tests pass.

Reply:
I did some testing, but I can't replicate the error with the `MySQL5InnoDBDialect` any longer. :confused:

However, I _did_ confirm that the shorthand notation (`"MySQL"` or `"mysql"`) works great on both engines.

I pushed a commit to update the dialect to `"MySQL"` in both the `.env.template` (which feeds the main `Application.cfc`) and the tests Application.cfc. This should future-proof that dialect setting.